src/python/pants/subsystem/subsystem.py

```diff
@@ -169,3 +174,33 @@ def get_options(self):
     :API: public
     """
     return self._scoped_options
+
+  @staticmethod
+  def get_streaming_workunit_callbacks(subsystem_names: List[str]) -> List[Callable]:
```

Comment:
This method does something somewhat complex -- might it be useful to add a docstring?